systems/controllers/controlUtil.h

```diff
@@ -40,6 +40,13 @@ typedef struct _support_state_element
   bool support_logic_map[4];
 } SupportStateElement;
 
+struct DrakeRobotState {

```

Comment:
why did you do this?  was it primarily to stop carrying around the template cases?